### ISSUE TITLE
Error "No such built-in module: node:timers/promise"

### ISSUE THREAD

**Issue by MartinWolfert** [sitespeedio/sitespeed.io#4008]
### Your question

Hi @Sitespeedio,

since a few days I get the following error running sitespeedio:

_sitespeedio@sitespeed:~$ /opt/sitespeed/bin/run-sitespeed-cron-somedomain.sh
internal/process/esm_loader.js:74
    internalBinding('errors').triggerUncaughtException(
                              ^

Error [ERR_UNKNOWN_BUILTIN_MODULE]: No such built-in module: node:timers/promises
    at new NodeError (internal/errors.js:322:7)
    at Loader.builtinStrategy (internal/modules/esm/translators.js:289:11)
    at new ModuleJob (internal/modules/esm/module_job.js:63:26)
    at Loader.getModuleJob (internal/modules/esm/loader.js:258:11)
    at async ModuleWrap.<anonymous> (internal/modules/esm/module_job.js:78:21)
    at async Promise.all (index 1)
    at async link (internal/modules/esm/module_job.js:83:9) {
  code: 'ERR_UNKNOWN_BUILTIN_MODULE'
}_

The same error occurs with "sitespeed.io -v"
Via npm, the sitespeedio version is 9.9.2

_npm -v sitespeed.io
9.9.2_

Any ideas .... how to fix this?

Thy and best,
Martin

**Comment by MartinWolfert** [comment]
Hi @soulgalore,
thanks for your reply.
I have installed nodes version 16.20.2.

The funny thing: I reinstalled all the stuff (on Linux)  as it is described on this page: https://www.sitespeed.io/documentation/sitespeed.io/installation/ and sitespeed is fully functional again.

I will try updating nodes next week, because I need the actual running sitespeed for a customer demo on Saturday :)

I will come back to this topic after updating nodes next week.

Best,
Martin
